internal/app/init/internal/rootfs/mount/mount.go

```diff
@@ -130,6 +132,24 @@ func (i *Initializer) InitOwned() (err error) {
 	return nil
 }
 
+// ExtraDevices mounts the extra devices.
+func ExtraDevices(data *userdata.UserData) (err error) {
+	if data.Install == nil || data.Install.ExtraDevices == nil {
+		return nil
+	}
+	for _, extra := range data.Install.ExtraDevices {
+		for i, part := range extra.Partitions {
```

Comment:
last nit, do we want to throw an error here if the mount point is not under `/var`?

Reply:
Ahh good point. Should be simple enough 👍.

Reply:
Gonna make this happen in a followup PR. I would like to get this in so we can use it for our storage.